src/config/settings.js

```diff
@@ -221,7 +221,18 @@ export default {
     hideWindowTitle: false, // Configure if the window title is shown in the window title bar or not
     sideBarOpenByDefault: false, // Configure if the sidebar (and its content panel) is open by default
   },
-  windows: [], // Array of windows to be open when mirador initializes (each object should at least provide a loadedManifest key with the value of the IIIF presentation manifest to load)
+  windows: [ // Array of windows to be open when mirador initializes (each object should at least provide a manifestId key with the value of the IIIF presentation manifest to load)
+    /**
+    Example Window:
+    {
+      manifestId: 'https://iiif.harvardartmuseums.org/manifests/object/299843',
+      canvasIndex: 2,
+      thumbnailNavigationPosition: 'far-bottom',
+    }
+    // https://github.com/ProjectMirador/mirador/blob/master/src/state/actions/window.js#L48-L65
+    // https://github.com/ProjectMirador/mirador/blob/master/src/lib/MiradorViewer.js#L53-L62
```

Comment:
These links are bound to change.  Should we use the permalink syntax instead?

Reply:
Er. Well... I guess the commit that gets permalinked will also get out of date. Maybe it would be better to just link to the file and explicitly call out `defaultOptions` and `windowAction`?

Reply:
Thanks both of you for the suggestions. I've updated to use relative references to the path of the file and explicit variable names.